### PR TITLE
Documentation of Comments plugin configuration is more specific #8416

### PR DESCRIPTION
**Pull request by wszymanski**
### Context

Just minor improvement in the plugin configuration.

[skip changelog]

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature or improvement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Additional language file or change to the existing one (translations)

### Related issue(s):
1. #8416 

### Affected project(s):
- [x] `handsontable`
- [ ] `@handsontable/angular`
- [ ] `@handsontable/react`
- [ ] `@handsontable/vue`

### Checklist:

- [x] I have reviewed the guidelines about [Contributing to Handsontable](https://github.com/handsontable/handsontable/blob/master/CONTRIBUTING.md) and I confirm that my code follows the code style of this project.
- [ ] My change requires a change to the documentation.

### CODE REVIEW

**Review by Siemienik — APPROVED** [review]

**Review by Siemienik — COMMENTED** [review]
Could you additionally update the API Ref for the `9.0` version?

**Comment by wszymanski** [comment]
> Could you additionally update the API Ref for the `9.0` version?

Done.

File: docs/9.0/guides/cell-features/comments.md [hunk 1]
```diff
@@ -103,3 +103,62 @@ const hot = new Handsontable(container, {
 });
 ```
 :::
+
+## Set width and height for comment
```
Comment (kirszenbaum) [inline docs/9.0/guides/cell-features/comments.md:107]:
```suggestion
## Set a comment box's size
```

File: docs/9.0/guides/cell-features/comments.md [hunk 2]
```diff
@@ -103,3 +103,62 @@ const hot = new Handsontable(container, {
...
+
+You can use constant dimensions for comment box using `style` parameter.
```
Comment (kirszenbaum) [inline docs/9.0/guides/cell-features/comments.md:109]:
```suggestion
To set the width and height of a comment box, use the `style` parameter.
```

File: docs/9.0/guides/cell-features/comments.md [hunk 3]
```diff
@@ -103,3 +103,62 @@ const hot = new Handsontable(container, {
...
+    ['2018', 10, 11, 12, 13, 15, 16],
+    ['2019', 10, 11, 12, 13, 15, 16],
+  ],
+  rowHeaders: true,
+  colHeaders: true,
+  contextMenu: true,
+  comments: true,
+  cell: [
+    { row: 1, col: 1, comment: { value: 'Some comment' } },
+    { row: 2, col: 2, comment: { value: 'Comment 200x50 px', style: { width: 200, height: 50 } } }
```
Comment (kirszenbaum) [inline docs/9.0/guides/cell-features/comments.md:128]:
```suggestion
    { row: 1, col: 1, comment: { value: 'Some comment' } },
    // add the `style` parameter
    { row: 2, col: 2, comment: { value: 'Comment 200x50 px', style: { width: 200, height: 50 } } }
```

File: docs/9.0/guides/cell-features/comments.md [hunk 4]
```diff
@@ -103,3 +103,62 @@ const hot = new Handsontable(container, {
...
+    ['', 'Tesla', 'Toyota', 'Honda', 'Ford'],
+    ['2018', 10, 11, 12, 13, 15, 16],
+    ['2019', 10, 11, 12, 13, 15, 16],
+    ['2020', 10, 11, 12, 13, 15, 16],
+  ],
+  rowHeaders: true,
+  colHeaders: true,
+  contextMenu: true,
+  comments: { 
+    displayDelay: 100,
```
Comment (kirszenbaum) [inline docs/9.0/guides/cell-features/comments.md:155]:
```suggestion
  contextMenu: true,
  comments: { 
    // on mouseover, wait 2 seconds before the comment box displays
    displayDelay: 2000,
```

File: docs/9.0/guides/cell-features/comments.md [hunk 5]
```diff
@@ -103,3 +103,62 @@ const hot = new Handsontable(container, {
...
+    { row: 1, col: 1, comment: { value: 'Some comment' } },
+    { row: 2, col: 2, comment: { value: 'Comment 200x50 px', style: { width: 200, height: 50 } } }
+  ],
+  height: 'auto',
+  licenseKey: 'non-commercial-and-evaluation'
+});
+```
+:::
+
+## Configure display delay
```
Comment (kirszenbaum) [inline docs/9.0/guides/cell-features/comments.md:136]:
```suggestion
## Set a delay for displaying comments
```

File: docs/9.0/guides/cell-features/comments.md [hunk 6]
```diff
@@ -103,3 +103,62 @@ const hot = new Handsontable(container, {
...
+
+It's also possible to set how many time it will take to show a comment. Just use `displayDelay` parameter.
```
Comment (kirszenbaum) [inline docs/9.0/guides/cell-features/comments.md:138]:
```suggestion
To display comments after a pre-configured time delay, use the `displayDelay` parameter.
```

File: docs/next/guides/cell-features/comments.md [hunk 7]
```diff
@@ -103,3 +103,62 @@ const hot = new Handsontable(container, {
 });
 ```
 :::
+
+## Set width and height for comment
```
Comment (kirszenbaum) [inline docs/next/guides/cell-features/comments.md:107]:
```suggestion
## Set a comment box's size
```

File: docs/next/guides/cell-features/comments.md [hunk 8]
```diff
@@ -103,3 +103,62 @@ const hot = new Handsontable(container, {
...
+
+You can use constant dimensions for comment box using `style` parameter.
```
Comment (kirszenbaum) [inline docs/next/guides/cell-features/comments.md:109]:
```suggestion
To set the width and height of a comment box, use the `style` parameter.
```

File: docs/next/guides/cell-features/comments.md [hunk 9]
```diff
@@ -103,3 +103,62 @@ const hot = new Handsontable(container, {
...
+    ['2018', 10, 11, 12, 13, 15, 16],
+    ['2019', 10, 11, 12, 13, 15, 16],
+  ],
+  rowHeaders: true,
+  colHeaders: true,
+  contextMenu: true,
+  comments: true,
+  cell: [
+    { row: 1, col: 1, comment: { value: 'Some comment' } },
+    { row: 2, col: 2, comment: { value: 'Comment 200x50 px', style: { width: 200, height: 50 } } }
```
Comment (kirszenbaum) [inline docs/next/guides/cell-features/comments.md:128]:
```suggestion
    { row: 1, col: 1, comment: { value: 'Some comment' } },
    // add the `style` parameter
    { row: 2, col: 2, comment: { value: 'Comment 200x50 px', style: { width: 200, height: 50 } } }
```

File: docs/next/guides/cell-features/comments.md [hunk 10]
```diff
@@ -103,3 +103,62 @@ const hot = new Handsontable(container, {
...
+  ],
+  height: 'auto',
+  licenseKey: 'non-commercial-and-evaluation'
+});
+```
+:::
+
+## Configure display delay
```
Comment (kirszenbaum) [inline docs/next/guides/cell-features/comments.md:136]:
```suggestion
## Set a delay for displaying comments
```

File: docs/next/guides/cell-features/comments.md [hunk 11]
```diff
@@ -103,3 +103,62 @@ const hot = new Handsontable(container, {
...
+
+It's also possible to set how many time it will take to show a comment. Just use `displayDelay` parameter.
```
Comment (kirszenbaum) [inline docs/next/guides/cell-features/comments.md:138]:
```suggestion
To display comments after a pre-configured time delay, use the `displayDelay` parameter.
```

File: docs/next/guides/cell-features/comments.md [hunk 12]
```diff
@@ -103,3 +103,62 @@ const hot = new Handsontable(container, {
...
+    ['', 'Tesla', 'Toyota', 'Honda', 'Ford'],
+    ['2018', 10, 11, 12, 13, 15, 16],
+    ['2019', 10, 11, 12, 13, 15, 16],
+    ['2020', 10, 11, 12, 13, 15, 16],
+  ],
+  rowHeaders: true,
+  colHeaders: true,
+  contextMenu: true,
+  comments: { 
+    displayDelay: 100,
```
Comment (kirszenbaum) [inline docs/next/guides/cell-features/comments.md]:
```suggestion
  contextMenu: true,
  comments: { 
    // on mouseover, wait 2 seconds before the comment box displays
    displayDelay: 2000,
```

**Review by kirszenbaum — CHANGES_REQUESTED** [review]

**Review by aninde — APPROVED** [review]
Cool examples, they will make it easier for users for sure. In 5b402b851f2f4490da6be4eb312de4c4933015c5, I increased the delay a little so that the demo would do better. I think I should suggest changes instead of committing them right away. Should I withdraw them or can it stay that way?

**Comment by wszymanski** [comment]
> Cool examples, they will make it easier for users for sure. In [5b402b8](https://github.com/handsontable/handsontable/commit/5b402b851f2f4490da6be4eb312de4c4933015c5), I increased the delay a little so that the demo would do better. I think I should suggest changes instead of committing them right away. Should I withdraw them or can it stay that way?

Please do not commit to branch and revert it. Then I will apply similar suggestion done by @kirszenbaum .

**Review by kirszenbaum — APPROVED** [review]

**Review by aninde — APPROVED** [review]
👏

**Review by swistach — APPROVED** [review]
LGTM 👏 

PS. Sorry for the delay 🕙 👀

**Pull request merged**